### PR TITLE
Fix flameshot not exiting when it should

### PR DESCRIPTION
**Pull request by veracioux**

### CODE REVIEW

File: src/core/controller.h [hunk 1]
```diff
@@ -35,12 +35,9 @@ class Controller : public QObject
 
     Controller(const Controller&) = delete;
     ~Controller();
-    void operator=(const Controller&) = delete;
```
Comment (borgmanJeremy) [inline src/core/controller.h:38]: Why is this line removed? It seems like the copy assignment for this class might need to be explicitly marked delete or we should implement the copy/ assignment operators. I don't think the default ones will be correct. (Nor do I think this class should be assigned  or copied).

**Review by borgmanJeremy — CHANGES_REQUESTED** [review]

File: src/core/controller.h (hunk 1 above)
Comment (veracioux) [reply src/core/controller.h:38]: https://doc.qt.io/qt-5/qobject.html#no-copy-constructor-or-assignment-operator

Comment (borgmanJeremy) [reply src/core/controller.h:38]: Oh, thanks.. Today I learned.

**Review by borgmanJeremy — APPROVED** [review]
